From 6507099e7e33be32b31a6492beb440aafb9f18bd Mon Sep 17 00:00:00 2001
From: David Dixon <david.dixon@amd.com>
Date: Wed, 20 Aug 2025 19:47:26 +0000
Subject: [PATCH 10/16] Install `libhipblas_fortran.so`

This is required by the test and benchmark clients but was not part of
the installation so far.
---
 projects/hipblas/clients/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/projects/hipblas/clients/CMakeLists.txt b/projects/hipblas/clients/CMakeLists.txt
index e86eaa446b..af214c51fa 100644
--- a/projects/hipblas/clients/CMakeLists.txt
+++ b/projects/hipblas/clients/CMakeLists.txt
@@ -121,6 +121,7 @@ if( BUILD_CLIENTS_TESTS OR BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_SAMPLES )
       # Set Fortran module output directory
       set(CMAKE_Fortran_MODULE_DIRECTORY ${PROJECT_BINARY_DIR}/include/hipblas)
       add_library(hipblas_fortran OBJECT ${hipblas_f90_source})
+      rocm_install(TARGETS hipblas_fortran)
   endif()
 
   if( NOT WIN32 )
-- 
2.25.1

